### PR TITLE
Botup gtm

### PR DESCRIPTION
**Pull request by venkatkalluri-500apps**
Botup GTM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,47 @@
-# PushNinja
+# Botup
+[Get Started](https://infinity.500apps.com/Botup) with with [Botup](https://botup.com)
+
+A powerful platform to build chatbots for your teams on Facebook, Slack, WhatsApp, SMS, and Website.
+Understand user behavior on the website with event triggers for each user event like Create flow, create Bot etc and use these triggers to create powerful automation campaigns.
+As the customer base grows from various locations, there’s a wide scope for business growth. Botup has the capability to update all your operations and make sure that you reach your customers on time.
+Botup serves you the best with easy setup options and enables advanced features to never miss anything that your target audience wanted. Target the locations, capture the leads, and grow your network to the corners of the world.
+
+- [Get Started](https://infinity.500apps.com/Botup) with your web push notification delivery with [Botup](https://Botup.com)
```
Comment (subramaniyamkolluri) [inline README.md]: https://botup.com

https://infinity.500apps.com/botup

Comment (subramaniyamkolluri) [reply README.md]: What is push notification

File: README.md [hunk 2]
```diff
@@ -1 +1,47 @@
-# PushNinja
+# Botup
+[Get Started](https://infinity.500apps.com/Botup) with with [Botup](https://botup.com)
```
Comment (subramaniyamkolluri) [inline README.md]: /botup

File: README.md (hunk 1 above)
Comment (venkatkalluri-500apps) [reply README.md]: Changed Sir

File: README.md (hunk 2 above)
Comment (venkatkalluri-500apps) [reply README.md]: Changed sir

File: README.md [hunk 3]
```diff
@@ -1 +1,47 @@
...
+Click on the Triggering tile and select All pages from the tags list.
+After you add the trigger, hit the Save button on the right.
+Name your tag and click Save.
+Once you’re back in GTM’s home, click on the Publish button to apply the changes.
+
+    ### Support 
+      Contact our support team through “Live Chat” or “Help us improve” options through our    application.
+
+![alt text]
+(https://drive.ninjacapture.com/HYS2CsEMGNIBwC4FcBOBTAtPAjBgzNngBwYCs2A7HgHRxIBGANiNNaQAzvxwDO1A9igDmAehABbSELQiATOwAseIA?type=widget)
```
Comment (subramaniyamkolluri) [inline README.md]: Getting page not found

**Pull request merged**